spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb

```diff
@@ -120,17 +113,17 @@ def time_component
   end
 end
 
-# The Win32::TaskScheduler and PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task classes should be
+# The Win32::TaskScheduler and PuppetX::PuppetLabs::ScheduledTask::V1 classes should be
 # API compatible and behave the same way.  What differs is which Windows API is used to query
 # and affect the system.  This means for testing, any tests should be the same no matter what
 # provider or concrete class (which the provider uses) is used.
-klass_list = Puppet.features.microsoft_windows? ? [Win32::TaskScheduler, PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task] : []
```

Comment:
I think it might be best to do

```ruby
[PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task, PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task]
```

Then on the next commit it will become 

```ruby
[PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task, PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2Task]
```
Which will probably be more self-evident with respect to changing the api2 provider to use the other helper.